streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -68,6 +69,12 @@ public void shouldNotHaveNullStoreNameOnReduce() throws Exception {
         groupedStream.reduce(MockReducer.STRING_ADDER, storeName);
     }
 
+    @Test(expected = InvalidTopicException.class)
+    public void shouldNotHaveInvalidStoreNameOnReduce() throws Exception {
+        String storeName = "~foo bar~";
```

Comment:
Use a class member and use in all test methods to avoid code duplication.
Please add `final` keyword wherever possible.